ssas/groups_test.go

```diff
@@ -77,6 +78,43 @@ func (s *GroupsTestSuite) TestCreateGroup() {
 	assert.EqualError(s.T(), err, "group_id cannot be blank")
 }
 
+func (s *GroupsTestSuite) TestDeleteGroup() {
+	group := Group{GroupID: "groups-test-delete-group"}
+	err := s.db.Create(&group).Error
+	if err != nil {
+		s.FailNow(err.Error())
+	}
+
+	system := System{GroupID: group.GroupID, ClientID: "groups-test-delete-group"}
+	err = s.db.Create(&system).Error
+	if err != nil {
+		s.FailNow(err.Error())
+	}
+
+	key1Str := "publickey1"
+	encrKey1 := EncryptionKey{
+		SystemID: system.ID,
+		Body:     key1Str,
+	}
+	err = s.db.Create(&encrKey1).Error
+	if err != nil {
+		s.FailNow(err.Error())
+	}
+
+	key2Str := "publickey2"
```

Comment:
System:(active)EncryptKey should be a 1:1 relationship, so we probably shouldn't be testing with two active encryption keys.